cmd/tailscale/cli/funnel.go

```diff
@@ -20,7 +20,13 @@ import (
 	"tailscale.com/util/mak"
 )
 
-var funnelCmd = newFunnelCommand(&serveEnv{lc: &localClient})
+var funnelCmd = func() *ffcli.Command {
+	se := &serveEnv{lc: &localClient}
+	if os.Getenv("TAILSCALE_FUNNEL_V2") == "on" {
```

Comment:
Also add a little doc here about what this flag is/that it's currently unsupported. Could even link the GH ticket